src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/controllers/PersonControllerTest.kt

```diff
@@ -194,5 +194,13 @@ internal class PersonControllerTest(
         """.removeWhitespaceAndNewlines()
       )
     }
+
+    it("responds with a 404 NOT FOUND status when person isn't found") {
+      whenever(getAddressesForPersonService.execute(id)).thenReturn(null)
+
+      val result = mockMvc.perform(get("/persons/$id/addresses")).andReturn()
+
+      result.response.status.shouldBe(404)
```

Comment:
We should probably start using import org.springframework.http.HttpStatus for these as we have in our Smoke Tests. I've created https://dsdmoj.atlassian.net/browse/HIA-162 to deal with this though :)

Reply:
Good spot, I've updated!